js/html/Index.svelte

```diff
@@ -23,6 +23,21 @@
 	export let max_height: number | undefined = undefined;
 	export let container = false;
 	export let padding = true;
+
+	const has_visible_content = (html: string): boolean => {
+		if (!html?.trim()) return false;
+		if (typeof document === "undefined") return true;
+		const div = document.createElement("div");
+		div.innerHTML = html;
+		document.body.appendChild(div);
+		const result = Array.from(div.children).some(
+			(el) => el.checkVisibility?.() ?? true
+		);
+		document.body.removeChild(div);
+		return result;
+	};
```

Comment:
This feels a bit heavy-handed for just checking whether HTML content contains visible elements, i.e. we need to change the DOM which also might not work super well with SSR. 

Could we do something simpler to check if there's at least one non-meta, non-script element?

```
	return Array.from(div.children).some((el) => {
		const tag = el.tagName.toLowerCase();
		return !["meta", "script", "style", "link", "title"].includes(tag);
	});
```